src/test/java/com/saylorsolutions/fnstate4j/StateStoreGlobalTest.java

```diff
@@ -0,0 +1,32 @@
+package com.saylorsolutions.fnstate4j;
+
+import static org.junit.Assert.*;
+
+import java.lang.reflect.Field;
+
+import org.junit.Test;
+
+import com.saylorsolutions.fnstate4j.StateStore.Global;
+
+public class StateStoreGlobalTest {
+	@Test
+	public void sameInstance() {
+		assertSame(StateStore.Global.instance(), StateStore.Global.instance());
+	}
+
+	@Test(expected = IllegalAccessException.class)
+	public void resistsNewInstanceReflection() throws IllegalArgumentException, IllegalAccessException {
+		Class<StateStore.Global> global = StateStore.Global.class;
+		final Field[] holderFields = global.getDeclaredFields();
+		holderFields[0].setAccessible(true);
+		holderFields[0].set(null, new StateStore());
+	}
```

Comment:
This is really a duplicate test once it's fixed to not try to set the holder field as if it's static.